### PR TITLE
Travis + Coveralls + Testability

### PR DESCRIPTION
**Pull request by jacobgreenleaf**

### CODE REVIEW

File: imageprocessor/processorcommand/runner.go [hunk 1]
```diff
@@ -35,8 +35,6 @@ func runProcessorCommand(command string, args []string) error {
 
 		return err
 	}
-
-	return nil
```
Comment (briankassouf) [inline imageprocessor/processorcommand/runner.go]: Do we not need to return at the end of this function?

File: imagestore/localstore.go [hunk 2]
```diff
@@ -27,17 +28,9 @@ func (this *LocalImageStore) Exists(obj *StoreObject) (bool, error) {
...
-func (this *LocalImageStore) Save(src string, obj *StoreObject) (*StoreObject, error) {
-	// open input file
-	fi, err := os.Open(src)
-	if err != nil {
-		return nil, err
-	}
-
-	defer fi.Close()
-
+func (this *LocalImageStore) Save(src io.Reader, obj *StoreObject) (*StoreObject, error) {
```
Comment (briankassouf) [inline imagestore/localstore.go]: :+1:

File: main.go [hunk 3]
```diff
@@ -12,5 +14,13 @@ func main() {
 
 	config := mandibleConf.NewConfiguration(configFile)
 	server := mandible.CreateServer(config)
-	server.Start()
+	muxer := http.NewServeMux()
+	server.Configure(muxer)
+
+	port := ":" + os.Getenv("PORT")
+	if port == ":" {
+		port = fmt.Sprintf(":%d", server.Config.Port)
```
Comment (briankassouf) [inline main.go]: this could just use config.Port ay?

File: imageprocessor/processorcommand/runner.go (hunk 1 above)
Comment (jacobgreenleaf) [inline imageprocessor/processorcommand/runner.go]: unreachable

**Pull request merged**